### ISSUE TITLE
serialization of a getter

### ISSUE THREAD

**Issue by oak-tree** [FasterXML/jackson-datatype-hibernate#60]
Hey there, in version `2.2.3` I get all my getters that not mark with `JSON ignore` to be serialized. But, when the lastest version `2.4.2`. the getters that do not have  any `property` related to them don't get `serialized`.

Code example:
============

	@LastModifiedBy
        @ManyToOne(optional = true)
        @JoinColumn(name="MODIFIER_ID", referencedColumnName = "ID")
         private User modifiedBy;

	public String getModifierName() {
		if (modifiedBy == null) {
			return "";
		}
		return modifiedBy.getFirstName() + " " + modifiedBy.getLastName();
	}

To sum up:
---------------
In `2.2.3` I will have `modifierName` field in the json out. But, in `2.4.2` I wont get this kind of field
